libs/zauth/src/Data/ZAuth/Creation.hs

```diff
@@ -96,6 +99,22 @@ accessToken1 dur usr = do
     d <- liftIO $ asGenIO (uniform :: GenIO -> IO Word64) g
     accessToken dur usr d
 
+legalHoldUserToken :: Integer -> UUID -> Word32 -> Create (Token LegalHoldUser)
+legalHoldUserToken dur usr rnd = do
+    d <- expiry dur
+    newToken d LU Nothing (mkLegalHoldUser usr rnd)
+
+legalHoldAccessToken :: Integer -> UUID -> Word64 -> Create (Token LegalHoldAccess)
+legalHoldAccessToken dur usr con = do
+    d <- expiry dur
+    newToken d LA Nothing (mkLegalHoldAccess usr con)
+
+legalHoldAccessToken1 :: Integer -> UUID -> Create (Token LegalHoldAccess)
+legalHoldAccessToken1 dur usr = do
```

Comment:
what's the idea here?  (i know, same as `accessToken1` above, but could you add a haddock line in both places even though it's not related?  :))

Reply:
Good idea. Done.